app/templates/admin/createSingleEvent.html

```diff
@@ -166,10 +166,10 @@ <h1 id="title">{{page_title}}</h1>
           </table>
         </div>
       </div>
-      <div class="col-md-6 form-group">
+      <div style="margin: -25px 0px 0px 0px" class="col-md-6 form-group">
```

Comment:
This inline style messes up the appearance when creating an event that isn't the all-volunteer event.